app/package.json

```
@@ -0,0 +1,62 @@
+{
+  "name": "@dcgrants/app",
+  "version": "0.0.1",
+  "private": true,
+  "scripts": {
+    "clean": "rimraf dist",
+    "dev": "vite",
+    "build": "vue-tsc --noEmit && vite build",
+    "serve": "vite preview",
+    "test:unit": "vue-cli-service test:unit",
+    "test:e2e": "vue-cli-service test:e2e",
+    "lint": "eslint --ext .ts,.js,.vue .",
+    "precommit": "lint-staged",
+    "prettier": "prettier --write"
```

This command doesn't work! Options to fix are:
```suggestion
    "prettier": "prettier --write ."
```

OR 

```suggestion
    "prettier": "prettier --write \"**/*.{vue,json,md,sol,js,ts}\""
```